remoting/connection_test.go

```diff
@@ -2,13 +2,15 @@ package remoting
 
```

Comment:
Can we have a test which uses an enabled TLS config?

Reply:
Done!